easybuild/easyconfigs/o/OrthoFinder/OrthoFinder-2.3.8-intel-2019b-Python-3.7.4.eb

```diff
@@ -0,0 +1,47 @@
+easyblock = 'Tarball'
+
+name = 'OrthoFinder'
+version = '2.3.8'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/davidemms/OrthoFinder'
+description = "OrthoFinder is a fast, accurate and comprehensive platform for comparative genomics"
+
+toolchain = {'name': 'intel', 'version': '2019b'}
+
+# https://github.com/davidemms/OrthoFinder
+github_account = 'davidemms'
+source_urls = [GITHUB_SOURCE]
+sources = ['%(version)s.tar.gz']
+patches = [
+    '%(name)s-%(version)s_inject_python_shebang.patch',
```

Comment:
patch is no longer needed thanks to your fix in https://github.com/easybuilders/easybuild-framework/pull/3183?